app/components/select-year/template.hbs

```diff
@@ -1 +1 @@
-{{primitive-select name=name items=years update=attrs.update prompt=prompt}}
+{{primitive-select name=name data-stripe=data-stripe items=years update=attrs.update prompt=prompt}}
```

Comment:
Same as above, I think we can remove `name=` given the other change to primitive select.